src/styles/normalize.scss

```diff
@@ -1,24 +1,28 @@
 /*! normalize.css v3.0.2 | MIT License | git.io/normalize */
 
-/**
- * 1. Set default font family to sans-serif.
- * 2. Prevent iOS text size adjust after orientation change, without disabling
- *    user zoom.
- */
-
-html {
-  font-family: sans-serif; /* 1 */
-  -ms-text-size-adjust: 100%; /* 2 */
-  -webkit-text-size-adjust: 100%; /* 2 */
-}
-
-/**
- * Remove default margin.
- */
-
-body {
-  margin: 0;
-}
+//
+// Need to prevent any impact on html/body elements by react-cm-ui library
+// so that the library works correctly in widgets
+//
+// /**
+//  * 1. Set default font family to sans-serif.
+//  * 2. Prevent iOS text size adjust after orientation change, without disabling
+//  *    user zoom.
+//  */
+
+// html {
+//   font-family: sans-serif; /* 1 */
+//   -ms-text-size-adjust: 100%; /* 2 */
+//   -webkit-text-size-adjust: 100%; /* 2 */
+// }
+
+// /**
+//  * Remove default margin.
+//  */
+
+// body {
+//   margin: 0;
+// }
```

Comment:
Don't we still need to apply this somewhere?

Reply:
@groberts314 I moved the html-styles into the `.hc-ui` class, hc-admin `<body/>` already has `margin: 0;`